### ISSUE TITLE
Consider providing link to WCAG doc in terminal.integrated.minimumContrastRatio setting description

### ISSUE THREAD

**Issue by RMacfarlane** [microsoft/vscode#86144]
Testing https://github.com/microsoft/vscode/issues/85966

The setting description currently provides examples that reference WCAG compliance levels. It might be helpful to include a link to documentation about this standard.

**Comment by gregvanl** [comment]
Should have unique description text for the two example levels 4.5 and 7. https://github.com/microsoft/vscode/issues/86426
